code/src/components/TodoList.js

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import styled from 'styled-components/macro';
+import JSConfetti from 'js-confetti';
+
+import todos from '../reducers/todos';
+import { StyledDeleteButton, CheckAllButton } from './StyledButtons';
+import { StyledCheckbox } from './StyledCheckbox';
+import emptystate from '../assets/emptystate.png';
+
+const StyledList = styled.section`
+  &.display-items {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    margin: 10px auto;
+    height: 260px;
+    overflow-y: scroll;
+  }
+  &.no-items {
+    background-image: url(${emptystate});
+    height: 400px;
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: 60%;
+    object-fit: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  .list-item {
+    display: grid;
+    grid-template-columns: 9vw auto 15vw;
+    align-items: center;
+    padding: 0.5vh 2vw;
+    min-height: 10vh;
+    @media (min-width: 768px) {
+      grid-template-columns: 6vw auto 7vw;
+    }
+    @media (min-width: 1024px) {
+      grid-template-columns: 4vw auto 4vw;
+    }
+  }
+
+  .list-item:nth-child(2n + 1) {
+    background-color: #e6f3f5;
+  }
+
+  .list-item-info {
+    grid-column: 2;
+  }
+
+  .list-item-info > p {
+    font-size: 16px;
+    margin: 2px auto;
+    word-break: break-word;
+  }
+
+  .list-item-info > p.checked {
+    text-decoration: line-through;
+  }
+
+  .list-item-info > p:nth-child(2n + 2) {
```

Comment:
I like this way of styling the two p tags without having to give them class names and deal with them separately. What does the word-break:break-word do?